packages/@sanity/portable-text-editor/src/editor/hooks/useSyncValue.ts

```diff
@@ -142,7 +142,24 @@ export function useSyncValue(
                   if (hasChanges && isValid) {
                     const validationValue = [value[currentBlockIndex]]
                     const validation = validateValue(validationValue, schemaTypes, keyGenerator)
-                    if (validation.valid) {
+                    // Resolve validations that can be resolved automatically, without involving the user (but only if the value was changed)
+                    if (
+                      !validation.valid &&
+                      validation.resolution?.autoResolve &&
+                      validation.resolution?.patches.length > 0
+                    ) {
+                      // Only apply auto resolution if the value has been populated before and is different from the last one.
+                      if (!readOnly && previousValue.current && previousValue.current !== value) {
+                        // Give a console warning about the fact that it did an auto resolution
+                        console.warn(
+                          `${validation.resolution.action} for block with _key '${validationValue[0]._key}'. ${validation.resolution?.description}`,
```

Comment:
Should `${validation.resolution.action}` be `${validation.resolution?.action}` ?